### ISSUE TITLE
AWACS coverage for Ranges 31-32-33 (34?)

### ISSUE THREAD

**Issue by 59thLefty** [132nd-vWing/TRMA#58]
Can we please add an AWACS to cover at least the entirety of the drone Range 33, and the most of the rest of the sea ranges please? An E-3, racetrack on the east side of R33 (P3, P4) would cover almost all four ranges, and would be "behind" friendly fighters to be covered from the drones. The idea of an E-2 was talked about, but for the sake of coverage I think an E-3 would be better. If we stick to E-2, one above CVOA can cover R-31-32-33 but not R34 for sure (datalink transmit distance is also a factor, I think it is 180NM for an E-2)
Thanks!

**Comment by 132nd-Entropy** [comment]
yeah, I full agree. However, let's wait for @132ndNeck's return so he can make the decision

**Comment by 132ndNeck** [comment]
In the .miz there is a E-2 now (not sure who put it there), but I do not see any reason why it need to be a E-2 if a E-3 have better coverage and can cover all of the ranges better. 

Unless anyone have any objections, I suggest to change from E-2 to a E-3 and increase the altitude to 39.000ft to have best possible coverage?

**Comment by 132ndNeck** [comment]
I have added a E-3 (called OVERLORD 2) to the .miz, master comms file, and CombatFlite. Located in the CVOA at 39.000ft

@132nd-Entropy: Can I delete the E-2 Overlord, or will it cause any scripting errors?

**Comment by 132nd-Entropy** [comment]
It can't be deleted without the Navy group script being updated. We need awacs coverage over range 33, which was not covered by the awacs we had. Thought having a carrier awacs would add nice flavor but I can take it out of course

**Comment by 132ndNeck** [comment]
> It can't be deleted without the Navy group script being updated. We need awacs coverage over range 33, which was not covered by the awacs we had. Thought having a carrier awacs would add nice flavor but I can take it out of course

Are there any benefits of keeping the E-2? It will not be visual to anyone as I assume it will not takeoff/land on the carrier? Are there functions that only the E-2 can do for the carrier operations?

If not, then I suggest replacing it with the OVERLORD that I have prepared in the .miz, as the E-3 have longer range, and with it all ranges should be covered.

This issue is also related to #51 
Once that is in place so the AWACS can be activated when needed via F10, then the E-2 can be taken out (unless there is a argument for keeping it in place, ref my questions above)

**Comment by 59thLefty** [comment]
Performance of the radar and data link is the exact same for both E2 and E3, tested on Tu95 + MiG29 targets (RCS does not seem to matter on these, the MiG did not appear later on timber) 
RADAR range 180NM (ages ago it was a lot more for sure) 
Link transmit range 230NM 
We mainly would use this AWACS for R31, 32,33 but not for 34 in my opinion. That's a complex scenario range, which probably would need handling of the awacs track anyway (correct me if I am wrong). 
Furthest point of overlord track is 170NM  from drone range R33 NW so it is perfect. 
Furthest point from R31SW is 225NM, so even an E3 can not cover all ranges 31,32,33 from CVOA. Let me know if I can assist any further.

**Comment by 59thLefty** [comment]
Performance of the radar and data link is the exact same for both E2 and E3, tested on Tu95 + MiG29 targets (RCS does not seem to matter on these, the MiG did not appear later on timber) 

RADAR detection range seem to be a bottleneck of 
AWACS altitude, but the E2 caps at 30, 800 feet. So an E3 can cover more. 
Link transmit range 230NM 

We mainly would use this AWACS for R31, 32,33 but not for 34 in my opinion. That's a complex scenario range, which probably would need handling of the awacs track anyway (correct me if I am wrong). 
Furthest point of overlord track is 170NM  from drone range R33 NW so it is perfect. 
Furthest point from R31SW is 225NM, so even an E3 can not cover all ranges 31,32,33 from CVOA. Let me know if I can assist any further.

**Comment by 132nd-Entropy** [comment]
> > It can't be deleted without the Navy group script being updated. We need awacs coverage over range 33, which was not covered by the awacs we had. Thought having a carrier awacs would add nice flavor but I can take it out of course
> 
> Are there any benefits of keeping the E-2? It will not be visual to anyone as I assume it will not takeoff/land on the carrier? Are there functions that only the E-2 can do for the carrier operations?
> 
> If not, then I suggest replacing it with the OVERLORD that I have prepared in the .miz, as the E-3 have longer range, and with it all ranges should be covered.
> 
> This issue is also related to #51 Once that is in place so the AWACS can be activated when needed via F10, then the E-2 can be taken out (unless there is a argument for keeping it in place, ref my questions above)

Hi @132ndNeck as I said, no, to me it didnt matter. We needed an AWACS near the carrier so I thought adding a carrier E2 was the logical choice. As I said, I dont mind either way, so we take out the E2 and replace with the E3 you put in, np :)

**Comment by 132nd-Entropy** [comment]
closing this, as the E3 was implemented to cover those ranges.
